<launch>
  <arg name="config-file"/>
  <arg name="node-name"/>

  <rosparam command="load" file="$(arg config-file)"/>

  <node pkg="iai_qb_cube_driver" type="iai_qb_cube_interpolator" 
        name="$(arg node-name)" output="screen">
    <remap from="~out_command" to="/iai_qb_cube_driver/command"/>
    <remap from="~in_state" to="/joint_states"/>
    <remap from="~in_command" to="~command"/>
  </node>
</launch>
